blog/BLOG.md

```diff
@@ -50,24 +50,31 @@ Cerberus is a collection of the following tasks that work together to achieve a
 
 ## Where to go from here?
 
-Cerberus is compatible with any project that uses [fastlane] or [gradle] and [Jira] as the project tracker. If you would like to give it a spin:
+Cerberus is compatible with any project that uses [Fastlane] or [Gradle] and [Jira] as the project tracker. Here are some links if you would like to give it a spin:
 
-! Add pending links and instructions here!
+**Fastlane**
 
+- [Project Page](https://github.com/outware/fastlane-plugin-cerberus)
+- [Integration Instructions](https://github.com/outware/fastlane-plugin-cerberus/blob/develop/INTEGRATION.md)
+
+**Gradle**
+
+- [Project Page](https://github.com/outware/gradle-plugin-cerberus)
+- [Integration Instructions](https://github.com/outware/gradle-plugin-cerberus)
 
 ## Conclusion
 
 Cerberus helped us by 'wiring up' tools that wouldn't normally speak to each other. Jira now serves as a source of truth and as a dashboard of information. Our team members don't have to chase up on our tools or chase up each other, giving them time to think about and solve harder problems.
 
 ---
 
-Do you see an issue with our plugins or want to add a new feature? Feel free to submit a PR to Github <!link pending>
+Do you see an issue with our plugins or want to add a new feature? Feel free to submit a PR to Github
```

Comment:
Were you going to link the PRs?

Should we have a full stop or exlcamation mark here 

Feel free to submit a PR to Github**!**

Reply:
Fixed in 1658f51